services/api-db/docker-entrypoint-initdb.d/01-migrations.sql

```diff
@@ -1094,6 +1094,24 @@ CREATE OR REPLACE PROCEDURE
   END;
 $$
 
+CREATE OR REPLACE PROCEDURE
+  update_unique_environment_fact()
+
+  BEGIN
+    IF NOT EXISTS(
+      SELECT NULL
+      FROM INFORMATION_SCHEMA.COLUMNS
+      WHERE
+        table_name = 'environment_fact'
+        AND table_schema = 'infrastructure'
```

Comment:
Hey! I don't think this is doing what we need it to do.
If you run a `show index from environment_fact` you'll see that the unique index's name we're interested in is called "environment"
So we want to look for `environment`, and if we find it drop it, 
i.e. 
ALTER TABLE environment_fact DROP INDEX environment;
Then, let's name the new index as well, by creating it something like this
ALTER TABLE environment_fact ADD CONSTRAINT environment_fact_unique UNIQUE(environment, name, source);

Reply:
Did it :)